### ISSUE TITLE
3.0.172+: $page->status() is broken (up to current master/dev)

### ISSUE THREAD

**Issue by incognitoms** [processwire/processwire-issues#1441]
### Short description of the issue

$page->status() returns the page itself.

### Expected behavior

$page->status() should return an integer (bitmask). [See docs](https://processwire.com/api/ref/page/status/)

### Actual behavior

We upgraded some sites to the latest master (3.0.184) and were surprised to see that $page->status() no longer worked as it did before with 3.0.165 (old master).

We went through the commits and found that the issue was introduced in:

commit 7fe8e5f1fb1eca16a87e629373ebe71f102eb310
Author: Ryan Cramer <ryan@processwire.com>
Date:   Fri Feb 5 11:28:48 2021 -0500

Add new $pages->findJoin() method and $pages->find("selector, field=title|summary"); option that lets you specify which fields to autojoin in a page finding operation. Also added is an experimental PagesLoader::findMin() method that finds and loads pages in one query rather than splitting them out into separate find() and getById() calls. For now, we just use this method to handle the findJoin() and joinFields options, but may expand its use later. This commit also contains numerous performance optimizations to the Page class, and several updates to the PageFinder class to support the new autojoin options.

### Steps to reproduce the issue

Create a template as follows:
```
<?php

namespace ProcessWire;

$p = $pages->get(27); // default 404 page for example

// PW 3.0.165: the following returns int(1035), as expected:
var_dump($p->status());

/* With PW 3.0.172+, this is returned:

object(ProcessWire\Page)#168 (7) {
  ["id"]=>
  int(27)
  ["name"]=>
  string(7) "http404"
  ["parent"]=>
  string(1) "/"
  ["status"]=>
  string(8) "systemID"
  ["template"]=>
  string(10) "basic-page"
  ["title"]=>
  string(13) "404 Not Found"
  ["data"]=>
  array(1) {
    ["title"]=>
    string(13) "404 Not Found"
  }
}

*/
```

**Comment by ryancramerdesign** [comment]
Thanks @incognitoms I saw your report in my email last week and fixed in while working on some other changes. I don't think I remembered to reference this issue report, but it should be in last week's commits.

**Comment by incognitoms** [comment]
Thank you very much, Ryan!

For reference: the fix came with commit c02a9d6f4db0c52e8758deb6b04c34846f171d26
